components/ui/pathogen-map/esm-maps.tsx

```diff
@@ -0,0 +1,46 @@
+import { ArboContext } from "@/contexts/arbo-context/arbo-context";
+import { useContext } from "react";
+import { Layer, Source } from "react-map-gl";
+
+type Esms = "zika" | "dengue2015" | "dengue2050";
+
+const esmIds: Record<Esms, string> = {
+  zika: "zika_esm_source",
+  dengue2015: "dengue_2015_esm_source",
+  dengue2050: "dengue_2050_esm_source",
+};
+
+const esmUrls: Record<Esms, string> = {
+  zika: "mapbox://serotracker.7qe2zkgd",
+  dengue2015: "mapbox://serotracker.5wobnon2",
+  dengue2050: "mapbox://serotracker.9lzerqbc",
```

Comment:
Where did these urls come from?

Reply:
I got the rasters from yannik, converted them to 8bits and then uploaded them to mapbox giving us these links